tools/codegen/dest/ts_lowering.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Optional, Union, Tuple
```

Comment:
do you think this should be prefixed with `lazy_` just so its clear which files in `codegen/dest` are relevant to LTC?